packages/test/src/integration-tests.ts

```diff
@@ -708,7 +708,7 @@ export function runIntegrationTests(codec?: PayloadCodec): void {
       workflowType: 'returnWorkflowInfo',
       workflowId,
       historyLength: 3,
-      unsafe: { isReplaying: false },
+      unsafe: { isReplaying: false, now: Date.now },
```

Comment:
How does this work? the function isn't going to be serialized

Reply:
I don't imagine this test passes, will update